stripe/src/main/java/com/stripe/android/view/ShippingInfoWidget.kt

```diff
@@ -40,7 +40,7 @@ class ShippingInfoWidget @JvmOverloads constructor(
             field = value
 
             renderLabels()
-            countryAutoCompleteTextView.selectedCountry?.let(::updateConfigForCountry)
+            countryAutoCompleteTextView.selectedCountryCode?.let(::updateConfigForCountry)
```

Comment:
I could update this so that countryAutoCompleteTextView still has a selectedCountry and it is just resolve that from the country code each time.  That would minimize the impact to this class.

Reply:
Similarly for the other changes in this file.